core/trino-parser/src/main/java/io/trino/sql/ExpressionFormatter.java

```diff
@@ -727,17 +726,15 @@ private String visitFilter(Expression node, Void context)
         @Override
         protected String visitQuantifiedComparisonExpression(QuantifiedComparisonExpression node, Void context)
         {
-            return new StringBuilder()
-                    .append("(")
-                    .append(process(node.getValue(), context))
-                    .append(' ')
-                    .append(node.getOperator().getValue())
-                    .append(' ')
-                    .append(node.getQuantifier().toString())
-                    .append(' ')
-                    .append(process(node.getSubquery(), context))
-                    .append(")")
-                    .toString();
+            return "(" +
+                    process(node.getValue(), context) +
+                    ' ' +
+                    node.getOperator().getValue() +
+                    ' ' +
+                    node.getQuantifier().toString() +
+                    ' ' +
+                    process(node.getSubquery(), context) +
+                    ")";
```

Comment:
would this read better with `"(%s %s %s %s)".formatted(....)`?

Reply:
That's what I suspected. But that means no `String#formatted` (added in Java 15). `String#format` it is!